reference/configuration.html.md.erb

```diff
@@ -291,15 +291,18 @@ Times are in milliseconds unless units are specified.
 
 ## The `services` sections
 
-The `services` sections configure the mapping of ports on the application to ports and services on Fly.io. These mappings determine how connections to the application will be handled on their journey from the Fly.io edge network to running Fly Apps.
+The `services` sections define how Fly Proxy connects requests to services running within Machines, via an app's public [Anycast](/docs/reference/services/) or private [Flycast](/docs/reference/private-networking/#flycast-private-load-balancing) address. They also configure additional settings tied to Fly Proxy behavior for a service. If a service only needs HTTP and HTTPS on standard ports, you can use the less verbose [`http_service`](#the-http_service-section).
 
-You can have:
+A `services` section itself is a table of tables in TOML, so it is delimited with double square brackets. Tables within `[[services]]` configure additional features and behaviors for the service, and are described in subsequent sections. **At least one [`services.ports`](#services-ports) entry is required for a complete service definition, to set a port and handler for incoming requests.**
```

Comment:
I think we either leave it plain text or put it in a note. We should avoid using bold in this way.

Reply:
Oops! A whole coloured callout seemed like too big a deal and plain text too little, so I went with bold. Which do you think we should use?

(Is that a general convention, not to use bold inline?)

Adding it in plain text and moving the ports section up next to the top services section is more kinds of guidance than we had before. So plain text?

Reply:
```suggestion
A `services` section itself is a table of tables in TOML, so it is delimited with double square brackets. Tables within `[[services]]` configure additional features and behaviors for the service, and are described in subsequent sections. At least one [`services.ports`](#services-ports) entry is required for a complete service definition, to set a port and handler for incoming requests.
```